src/java.base/share/classes/sun/net/www/protocol/http/HttpURLConnection.java

```diff
@@ -1996,8 +1996,7 @@ private InputStream getInputStream0() throws IOException {
                 return inputStream;
             } while (redirects < maxRedirects);
 
-            throw new ProtocolException("Server redirected too many " +
-                                        " times ("+ redirects + ")");
+            throw new ProtocolException("Server redirected too many times ("+ redirects + ")");
```

Comment:
There's a missing space between " and + ; In order to avoid long line which are a pain to review in side diffs I would suggest to reformat and add a newline there, rather than a whitespace.

Reply:
@dfuch Thanks for the feedback! I updated as you suggested.